spec/models/events/new_comment_spec.rb

```diff
@@ -17,6 +17,11 @@
       Events::NewComment.publish!(comment)
     end
 
+    it 'marks the discussion reader as participating' do
```

Comment:
I basically tested participating enough to ensure that when someone
- creates a discussion
- creates a motion
- creates a comment
- creates a vote

then they are participating in the associated discussion.